src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+/* page region */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+/* end region */
+
+/* stars container region */
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  padding: 22px;
+}
+
+/* end region */
+
+/* stars region */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 2px;
+  padding-top: 0;
+}
+
+.stars--0 > .stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star:nth-child(n + 2) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(n + 3) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(n + 4) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars__star:last-child {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
just add these two rules to `.stars__star `
```
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
```
also use `background-position` to place image at the center
```suggestion
```